diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/comments/HeaderCommentRule.kt

```diff
@@ -172,10 +172,10 @@ class HeaderCommentRule(private val configRules: List<RulesConfig>) : Rule("head
                 node.addChild(PsiWhiteSpaceImpl(newLines), node.firstChildNode)
                 node.addChild(LeafPsiElement(BLOCK_COMMENT,
                     """
-                            /*
-                             * $copyrightText
-                             */
-                        """.trimIndent()),
+/*
+${handleMultilineCopyright(copyrightText)}
+*/
```

Comment:
You can align this string as it used to be, all leading spaces will be trimmed by `trimIndent`.